src/edu/csus/ecs/pc2/clics/API202003/JSONTool.java

```diff
@@ -69,11 +67,12 @@ public JSONTool(IInternalContest model, IInternalController controller) {
      *
      * @param submission
      */
+    @Override
     public ObjectNode convertToJSON(Run submission, HttpServletRequest servletRequest, SecurityContext sc) {
         ObjectNode element = mapper.createObjectNode();
-        element.put("id", getSubmissionId(submission));
-        element.put("language_id", getLanguageId(model.getLanguage(submission.getLanguageId())));
-        element.put("problem_id", getProblemId(model.getProblem(submission.getProblemId())));
+        element.put("id", IJSONTool.getSubmissionId(submission));
+        element.put("language_id", IJSONTool.getLanguageId(model.getLanguage(submission.getLanguageId())));
+        element.put("problem_id", IJSONTool.getProblemId(model.getProblem(submission.getProblemId())));
         element.put("team_id", new Integer(submission.getSubmitter().getClientNumber()).toString());
```

Comment:
The constructor `Integer(int)` is deprecated and slated for removal; the preferred alternative is the static factory method `Integer.valueOf(int)`.  (I'm aware this is old code, but if you happen to be editing this file it might be worth updating this.)

Reply:
Agree.  I fixed several of them in the same module and TeamService as well. 
 (next push).